wireapp/wire-android
# fix: revert of #3670 (WPB-14433)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-14433" title="WPB-14433" target="_blank"><img alt="Bug" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10803?size=medium" />WPB-14433</a>  [Android] Crash after longtapping on a conversation and putting phone to sleep
  </td></table>
  <br />
 



----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We introduced by caching the images, a problem visible as well by federation being enabled this week.

### Causes (Optional)

Image loader is cached, and tries to use federation for assets.

### Solutions

Revert the PR #3670 and pick the fixes needed only (#3689)

### Testing

#### Test Coverage (Optional)


#### How to Test

Login with 2 accounts in different backends (staging and prod) and you should NOT have failed to load images.

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
